db/migrate/20210601125525_ensure_ems_storage_features.rb

```diff
@@ -0,0 +1,35 @@
+class EnsureEmsStorageFeatures < ActiveRecord::Migration[6.0]
+  class MiqUserRole < ActiveRecord::Base
+    has_and_belongs_to_many :miq_product_features, :join_table => :miq_roles_features, :class_name => "EnsureEmsStorageFeatures::MiqProductFeature"
```

Comment:
If I recall, using a habtm ends up creating the intermediate join table class on the fly, but I can't remember if it can affect "real" updates. Do we use habtm elsewhere in migrations?

Reply:
I pulled this from https://github.com/ManageIQ/manageiq-schema/pull/564/files#diff-0572029e3d6c232fd7472ce434f3577bd44d06628727cbce2b250f3cf80b6f84R2-R4 which does similar updates to this